src/components/Search/SearchInput.jsx

```diff
@@ -26,7 +27,11 @@ const SearchInput = () => {
     <InputGroup
       className="u-mb-half-s u-mr-0-s u-mr-1"
       prepend={
-        <Icon className="u-pl-1" icon="magnifier" color={palette['coolGrey']} />
+        <Icon
+          className="u-pl-1"
+          icon={MagnifierIcon}
+          color={palette['coolGrey']}
```

Comment:
Not part of this PR but the palette['coolGrey'] here will not work in another theme.

Reply:
Bien vu, je nous rajoute une tâche de migration lente à ce sujet.